developers/weaviate/manage-data/read-all-objects.mdx

```diff
@@ -19,11 +19,11 @@ Weaviate provides the necessary APIs to iterate through all your data. This is u
 
 This is done with the help of the `after` operator, also called the [cursor API](../api/graphql/additional-operators.md#cursor-with-after).
 
-:::info Iterator 
+:::info Iterator
 The new API clients (currently supported by the Python Client v4), encapsulate this functionality as an `Iterator`.
 :::
 
-## Read all objects
+## Read object properties & ids
```

Comment:
Consider: 
Read object properties and ids

Reply:
Updated